neo/rawio/intanrawio.py

```diff
@@ -586,17 +766,81 @@ def read_rhd(filename):
         if len(channels_by_type[sig_type]) > 0:
             name = {4: "DIGITAL-IN", 5: "DIGITAL-OUT"}[sig_type]
             chan_info = channels_by_type[sig_type][0]
+            # So currently until we have get_digitalsignal_chunk we need to do a tiny hack to
+            # make this memory map work correctly. So since our digital data is not organized
+            # by channel like analog/ADC are we have to overwrite the native name to create
+            # a single permanent name that we can find with channel id 
             chan_info["native_channel_name"] = name  # overwite to allow memmap to work
             chan_info["sampling_rate"] = sr
-            chan_info["units"] = "TTL"  # arbitrary units so I did TTL for the logic
+            chan_info["units"] = "TTL"  # arbitrary units TTL for logic
             chan_info["gain"] = 1.0
             chan_info["offset"] = 0.0
             ordered_channels.append(chan_info)
-            data_dtype += [(name, "uint16", BLOCK_SIZE)]
+            if file_format == "header-attached":
+                data_dtype += [(name, "uint16", BLOCK_SIZE)]
+            elif file_format == 'one-file-per-signal':
+                data_dtype[sig_type] = "uint16"
+            else:
+                data_dtype[sig_type] += ["uint16"]
 
-    if bool(global_info["notch_filter_mode"]) and version >= V("3.0"):
-        global_info["notch_filter_applied"] = True
+    if global_info["notch_filter_mode"] == 2 and version >= V("3.0"):
+        global_info["notch_filter"] = '60Hz'
+    elif global_info["notch_filter_mode"] == 1 and version >= V("3.0"):
+        global_info["notch_filter"] = '50Hz'
     else:
-        global_info["notch_filter_applied"] = False
+        global_info["notch_filter"] = False
```

Comment:
Originally I added this code in because Intan was just returning the mode which isn't helpful. While I was at it I figured it would be even more helpful to say which notch filter was applied. But we could change this if it is unclear. I just wanted a clean way to represent that since neo was returning 0, 1, or 2 and so someone would need to go to the Intan documentation to know what these meant. Not relevant for this PR, but I want to make this fix now.